vagrant_resources/host_vms/auxiliary_disks/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
Optionally, we could instead add something like `*/auxiliary_disks/*` to the main .gitignore file. That could cover all future "auxiliary_disks" dirs as well, for future providers.